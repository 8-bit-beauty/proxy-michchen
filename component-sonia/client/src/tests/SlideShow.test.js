// import React from 'react';
// import { shallow } from 'enzyme';
// import RelatedItems from '../SlideShow.jsx';


// describe('RelatedItems', () => {
//   it('should render correctly in "debug" mode', () => {
//     const component = shallow(<RelatedItems debug />);
//     expect(component).toMatchSnapshot();
//   });
//   it('should render correctly with no props', () => {
//     const component = shallow(<RelatedItems />);
//     expect(component).toMatchSnapshot();
//   });
//   it('renders PzSlider component', () => {
//     const wrapper = shallow(
//       <RelatedItems />,
//     );
//     expect(wrapper).toMatchSnapshot();
//   });
// });
