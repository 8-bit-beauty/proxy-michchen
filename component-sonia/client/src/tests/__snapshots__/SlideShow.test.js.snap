// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RelatedItems renders PzSlider component 1`] = `
<div>
  <span
    className="page"
  >
    Page 
    1
     of 
    1
  </span>
  <button
    className="previous"
    onClick={[Function]}
    type="submit"
  >
    ‹
  </button>
  Loading...
  <button
    className="next"
    onClick={[Function]}
    type="submit"
  >
    ›
  </button>
</div>
`;

exports[`RelatedItems should render correctly in "debug" mode 1`] = `
<div>
  <span
    className="page"
  >
    Page 
    1
     of 
    1
  </span>
  <button
    className="previous"
    onClick={[Function]}
    type="submit"
  >
    ‹
  </button>
  Loading...
  <button
    className="next"
    onClick={[Function]}
    type="submit"
  >
    ›
  </button>
</div>
`;

exports[`RelatedItems should render correctly with no props 1`] = `
<div>
  <span
    className="page"
  >
    Page 
    1
     of 
    1
  </span>
  <button
    className="previous"
    onClick={[Function]}
    type="submit"
  >
    ‹
  </button>
  Loading...
  <button
    className="next"
    onClick={[Function]}
    type="submit"
  >
    ›
  </button>
</div>
`;
